Alpine-Transmission installation failure

### ✅ Have you read and understood the above guidelines?

yes

### 📜 What is the name of the script you are using?

alpine-transmission.sh

### 📂 What was the exact command used to execute the script?

bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/ct/alpine-transmission.sh)"

### ⚙️ What settings are you using?

- [ ] Default Settings
- [x] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Alpine

### 📝 Provide a clear and concise description of the issue.

Installing Transmissionsed: /var/lib/transmission/config/settings.json: No such file or directory

[ERROR] in line 20: exit code 1: while executing command sed -i '{s/"rpc-whitelist-enabled": true/"rpc-whitelist-enabled": false/g; s/"rpc-host-whitelist-enabled": true,/"rpc-host-whitelist-enabled": false,/g}' /var/lib/transmission/config/settings.json

/dev/fd/63: line 72: SPINNER_PID: unbound variable

### 🔄 Steps to reproduce the issue.

Start a normal installation with the following advanced configuration:

🧩  Using Advanced Settings on node moxy-2
  🖥️  Operating System: alpine
  🌟  Version: 3.21
  📦  Container Type: Unprivileged
  🔐  Root Password: ********
  🆔  Container ID: 115
  🏠  Hostname: alpine-transmission
  💾  Disk Size: 1 GB
  🧠  CPU Cores: 2
  🛠️  RAM Size: 4096 MiB
  🌉  Bridge: vmbr0
  📡  IP Address: 192.168.68.233/24
  🌐  Gateway IP Address: 192.168.68.1
  🚫  Disable IPv6: yes
  ⚙️  Interface MTU Size: Default
  🔍  DNS Search Domain: Host
  📡  DNS Server IP Address: Host
  🏷️  Vlan: Default
  📡  Tags: community-script;alpine;torrent;media
  🔑  Root SSH Access: yes
Warning: No SSH key provided.
  🔍  Verbose Mode: no
  🚀  Creating a Alpine-Transmission LXC using the above advanced settings

### ❌ Paste the full error output (if available).

Installing Transmissionsed: /var/lib/transmission/config/settings.json: No such file or directory

[ERROR] in line 20: exit code 1: while executing command sed -i '{s/"rpc-whitelist-enabled": true/"rpc-whitelist-enabled": false/g; s/"rpc-host-whitelist-enabled": true,/"rpc-host-whitelist-enabled": false,/g}' /var/lib/transmission/config/settings.json

/dev/fd/63: line 72: SPINNER_PID: unbound variable

### 🖼️ Additional context (optional).

Nothing special. I have a number of Helper-Script LXC containers running on a proxmox cluster. This is the first one that has resulted in an error.

--- a/install/alpine-transmission-install.sh
+++ b/install/alpine-transmission-install.sh
@@ -16,6 +16,7 @@ update_os
 msg_info "Installing Transmission"
 $STD apk add --no-cache transmission-cli transmission-daemon
 $STD rc-service transmission-daemon start
+sleep 5
 $STD rc-service transmission-daemon stop
 sed -i '{s/"rpc-whitelist-enabled": true/"rpc-whitelist-enabled": false/g; s/"rpc-host-whitelist-enabled": true,/"rpc-host-whitelist-enabled": false,/g}' /var/lib/transmission/config/settings.json
 msg_ok "Installed Transmission"

